src/cryfs/CMakeLists.txt

```diff
@@ -44,7 +44,13 @@ set(LIB_SOURCES
 
 add_library(${PROJECT_NAME} STATIC ${LIB_SOURCES})
 set_target_properties(${PROJECT_NAME} PROPERTIES OUTPUT_NAME cryfs)
-target_link_libraries(${PROJECT_NAME} PRIVATE cpp-utils blockstore blobstore fspp gitversion) # TODO Check that dependent projects don't get this linked in
```

Comment:
We can keep this here as a set of common libraries, and below only link the additional library if it's FreeBSD.